okapi-core/src/main/java/org/folio/okapi/service/impl/DockerModuleHandle.java

```diff
@@ -367,7 +367,7 @@ private int getExposedPort(JsonObject b) {
   @Override
   public Future<Void> start() {
     if (dockerPull) {
-      return pullImage().compose(x -> prepareContainer());
+      return pullImage().compose(x -> prepareContainer(), x -> prepareContainer());
```

Comment:
Maybe adding a line to explain why `x -> prepareContainer()` is called twice?

Reply:
It's not called twice.. Called on success and failure.. I'll use another more explicit method Future.recover.